getting-started/services.md

```diff
@@ -17,29 +17,28 @@ You use Timescale Console to create and manage Timescale services. Each service
 
 Timescale offers the following PostgreSQL service types:
 
-- [**Time-series and Analytics**](https://www.timescale.com/blog/what-is-a-time-series-database/#what-is-a-time-series-database)
-  : a Timescale DB instance optimized for your time-series and analytics workloads.
-- **All other workloads**: a [Dynamic PostgreSQL](https://www.timescale.com/dynamic-postgresql) instance where you select a compute range. You only pay for the base and the amount of extra CPU as you scale.
+- [**Time-series and Analytics**](https://www.timescale.com/blog/what-is-a-time-series-database/#what-is-a-time-series-database): a Timescale DB instance optimized for your time-series and analytics workloads.
+- **All other workloads**: a [Dynamic PostgreSQL](https://www.timescale.com/dynamic-postgresql) instance with a Dynamic Compute Range aligned to your business needs. You only pay for the base and the amount of extra CPU as we dynamically scale up and down between your min/max limits.
```

Comment:
```suggestion
- **All other workloads**: a [Dynamic PostgreSQL](https://www.timescale.com/dynamic-postgresql) instance with a Dynamic Compute range aligned to your business needs. You only pay for the base and the amount of extra CPU as we dynamically scale up and down between your min/max limits.
```

Reply:
At the moment Dynamic PostgreSQL is in early access pricing and users don't pay for the usage above the baseline

Reply:
Updated to say this.